deploy/cluster_role.yaml

```diff
@@ -13,6 +13,12 @@ rules:
   - create
   - list
   - watch
+- apiGroups:
```

Comment:
`subscriptions` are namespaced resources, so shouldn't you make this change in `deploy/role.yaml` instead?

Reply:
Yes, that was the first thing that I did, but then I noticed that the other permissions of `operators.coreos.com` are defined in the `cluster_role.yaml` file. I have it in my todo list to revisit it - if it is really needed. 
The only reasoning that I see is that the operator is going to be installed in one namespace (`openshift-operators`) but we need to manage subscriptions etc. also in other namespaces, so RoleBinding in one namespace won't help us. On the other hand, because it's gonna be installed in AllNamespaces mode, then we should get the permissions for all namespaces.
I just wanted to have it working and I kept the same way of defining the permissions as it was used before. But anyway, as I said - I want to revisit it. I can do it here or in separated PR

Reply:
that's confusing, because all API resources seem to be namespaced:
```
$ oc api-resources | grep "operators.coreos.com"
NAME                                  SHORTNAMES       APIGROUP                              NAMESPACED   KIND
catalogsourceconfigs                  csc              operators.coreos.com                  true         CatalogSourceConfig
catalogsources                        catsrc           operators.coreos.com                  true         CatalogSource
clusterserviceversions                csv,csvs         operators.coreos.com                  true         ClusterServiceVersion
installplans                          ip               operators.coreos.com                  true         InstallPlan
operatorgroups                        og               operators.coreos.com                  true         OperatorGroup
operatorsources                       opsrc            operators.coreos.com                  true         OperatorSource
subscriptions                         sub,subs         operators.coreos.com                  true         Subscription
packagemanifests                                       packages.operators.coreos.com         true         PackageManifest
```

Reply:
yes, sure. But before we limited the installation mode only to AllNamespaces, the user could install the operator in SingleNamespace, which means that the RoleBinding is created only inside of the namespace, thus the operator wouldn't be able to Get/Update the resources in other namespaces.
I know, I should have changed it in the "AllNamespace PR" but I didn't realize that at that time :man_shrugging: 
But as I said I can try it here ;-)

Reply:
ok, now I get why this change appears here ;)

Reply:
solved here: 5cea282